### DISCUSSION TITLE
GoPro11mini turns brick for help

### DISCUSSION THREAD

**Discussion by xzczcz** [gopro/labs#911] [Q&A]
My GoPro11Mini has become bricked since I upgraded the Labs firmware and can't record,Restore the official version of the firmware or the same problem,As long as you press the record to start recording for a few seconds, it gets stuck and reboots,After restarting, it will show that the file is being repaired,After fixing it, if you press the record button again, it will still get stuck and automatically restart,I don't know if it's a hardware problem or a firmware problem,I ruled out the problem with the memory card (not overheating to protect the temperature of the back is also very low)

**Comment by dnewman-gpsw** [comment]
Factory reset could help.  Unlikely to be a firmware issue as you have changed that twice.  Could be a hardware failure.

**Reply by xzczcz** [reply]
I tried to restore the factory settings, but it didn't work. Thank you for reminding me, maybe he really broke down because of the hardware problem, but I don't know what the problem is, and I haven't found anyone with the same problem as me.
